_saas-integrations/ga360/v1/ga360-v1.md

```diff
@@ -0,0 +1,160 @@
+    anchor: "retrieve-project-dataset-ids"
+    content: |
+      1. Login to Google and go to your Google Cloud Console.
+      2. To locate your list of projects, click on the navigation menu in the upper left-hand corner of the page.
+      3. Hover over **IAM & Admin** to pop open its menu. Select **Manage Resources**.
+      4. On this page, navigate to the project that includes the dataset you want to replicate data from and click on it.
+      5. Your **Project ID** and all of your **Dataset IDs** will be visible. Make note of the Project ID and Dataset ID that you want to replicate and keep it readily available for the {{ integration.display_name }} integration configuration page.
+
+  - title: "Create a GCP IAM service account"
+    anchor: "create-gcp-iam-service-account"
+    content: |
+      {% for substep in step.substeps %}
+      - [Step 2.{{ forloop.index }}: {{ substep.title }}](#{{ substep.anchor }})
+      {% endfor %}
+
+    substeps:
+      - title: "Define the service account details"
+        anchor: "define-service-account-details"
+        content: |
+          {% include destinations/google-bigquery/create-iam-service-account.html type="define-service-account-details" %}
+
+      - title: "Assign service account permissions"
+        anchor: "assign-service-account-permissions"
+        content: |
+          {% include destinations/google-bigquery/create-iam-service-account.html type="assign-bq-roles" %}
+
+      - title: "Create a JSON project key"
+        anchor: "create-json-project-key"
+        content: |
+          {% include destinations/google-bigquery/create-iam-service-account.html type="create-json-project-key" %}  
+            
+  - title: "add integration"
+    content: |
+      4. In the **GA 360 Dataset ID** and **GA 360 Project ID** fields, enter the Project and Dataset IDs you retrieved in [Step 1](#retrieve-project-dataset-ids).
+      5. In the **Service Account JSON** field, paste the JSON project key you obtained in [Step 2.3](#create-gcp-iam-service-account).
+  - title: "historical sync"
+  - title: "replication frequency"
+  - title: "track data" 
+
+
+# -------------------------- #
+#     Replication Details    #
+# -------------------------- #
+
+replication-sections:  
+  - content: |
+      While Stitch's {{ integration.display_name }} replicates the full tables for your dataset, tables are updated by the `ga_sessions_YYYYMMDD` non-field bookmark value. 
+
+      Daily, Google creates an export file, `ga_sessions_YYYYMMDD`. Intraday data is imported to the table `ga_sessions_intraday_YYYYMMDD` around three times per day. Once the final intraday import is complete, then the `ga_sessions_YYYYMMDD` table is complete and can be exported.
+
+      Stitch's {{ integration.display_name }} will re-replicate your data when Google exports a `ga_sessions_YYYYMMDD` table with the date portion greater than the previous export. Allow 24 hours to see complete updates of your data.
```

Comment:
I think this should say `replicate` instead of `re-replicate`. We only ever pull a table once.

If the sync gets interrupted while we're pulling a table, the tap will request that whole table again the next time, so some records will be re-replicated, but it's not the regular path.